components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -751,10 +751,15 @@ void MAMMicrophysics::run_impl(const double dt) {
 
         //
         auto invariants_icol = ekat::subview(invariants, icol);
-        //    auto cnst_offline_icol = ekat::subview(cnst_offline,icol);
-        //    mam4::mo_setinv::setinv(team, invariants_icol, atm.temperature,
-        //    atm.vapor_mixing_ratio,
-        //                            cnst_offline_icol, atm.pressure);
+        
+        view_1d cnst_offline_icol[mam4::mo_setinv::num_tracer_cnst];
+        for (int i = 0; i < mam4::mo_setinv::num_tracer_cnst; ++i) {
```

Comment:
Note that the order of the tracers in this interface is std::vector<std::string> var_names{"O3", "HO2", "NO3", "OH"};. See line 238.